src/Response.php

```diff
@@ -222,14 +222,6 @@ public function getBody(): Payload
         return $this->body;
     }
 
-    public function withPreviousResponse(?Response $previousResponse): self
-    {
-        $clone = clone $this;
-        $clone->previousResponse = $previousResponse;
-
-        return $clone;
-    }
```

Comment:
Why did you remove this?

Reply:
The method was duplicated somehow when I was rebasing.